### DISCUSSION TITLE
[Original YTM Player bug] Next song plays for a few seconds when repeat is on.

### DISCUSSION THREAD

**Discussion by inson1** [pear-devs/pear-desktop#1437] [Q&A]
When song repeat is enabled and the current song is about to end, instead of replaying the current song it plays the next song for a few seconds, and then switches back.

Is there a way to fix this?

**Comment by ArjixWasTaken** [comment]
I assume this happens on the yt music website itself?

**Reply by inson1** [reply]
@ArjixWasTaken 
I never used the website too much, but in pwa yes, even this app has this bug
Btw I said it is original youtube music problem :)

**Reply by ArjixWasTaken** [reply]
Yep, I figured that out, just wanted to make sure I understood correctly 😁
